hwdb.d/60-evdev.hwdb

```diff
@@ -921,3 +921,8 @@ evdev:input:b0003v172Fp0047*
 evdev:input:b0003v172Fp0505*
  EVDEV_ABS_00=::160
  EVDEV_ABS_01=::160
+
+# Waltop International Corp.  Batteryless Tablet
```

Comment:
The product mentioned in the commit title does not match with this comment.
Could you provide `/sys/class/input/eventX/device/id` ?

Reply:
Ah, you mentioned difference in the cover letter of the PR...
Please update the comment in the .hwdb file and the AIPTEK tablet is detected as Waltop one.

Reply:
Also, please drop doubled space between `Corp.` and `Batteryless`.